src/issue/screens/new-issue.screen.js

```diff
@@ -101,77 +109,91 @@ class NewIssue extends Component {
     return (
       <ViewContainer>
         {isPendingSubmitting && <LoadingModal />}
-        <ScrollView>
-          {repository.full_name &&
-            <ListItem
-              title={repository.full_name}
-              titleStyle={styles.titleSmall}
-              leftIcon={{
-                name: 'repo',
-                size: 17,
-                color: colors.grey,
-                type: 'octicon',
-              }}
-              hideChevron
-            />}
-          <SectionList title={translate('issue.newIssue.issueTitle', language)}>
-            <TextInput
-              underlineColorAndroid={'transparent'}
-              placeholder={translate('issue.newIssue.writeATitle', language)}
-              blurOnSubmit
-              multiline
-              onContentSizeChange={event =>
-                this.setState({
-                  issueTitleHeight: event.nativeEvent.contentSize.height,
-                })}
-              onChangeText={text => this.setState({ issueTitle: text })}
-              placeholderTextColor={colors.grey}
-              style={[
-                styles.textInput,
-                { height: Math.max(60, this.state.issueTitleHeight) },
-              ]}
-              value={issueTitle}
-            />
-          </SectionList>
-
-          <SectionList
-            title={translate('issue.newIssue.issueComment', language)}
-          >
-            <TextInput
-              underlineColorAndroid={'transparent'}
-              placeholder={translate('issue.newIssue.writeAComment', language)}
-              multiline
-              onChangeText={text => this.setState({ issueComment: text })}
-              onContentSizeChange={event =>
-                this.setState({
-                  issueCommentHeight: event.nativeEvent.contentSize.height,
-                })}
-              placeholderTextColor={colors.grey}
-              style={[
-                styles.textInput,
-                { height: Math.max(60, this.state.issueCommentHeight) },
-              ]}
-              value={issueComment}
-            />
-          </SectionList>
-
-          <SectionList>
-            <View style={styles.listItemContainer}>
+        <KeyboardAvoidingView
+          style={{ flex: 1 }}
+          behavior={'padding'}
+          keyboardVerticalOffset={Platform.select({
+            ios: 65,
```

Comment:
Why are the different values depending on the platform?

Reply:
The keyboardVerticalOffset is different because the height of the following vary by platform.
- React navigation headers.
- Height of the auto suggestion bar.
So I figured both platforms needs separate offset heights.